charts/victoria-metrics-cluster/README.md

```diff
@@ -12,9 +12,7 @@ Victoria Metrics Cluster version - high-performance, cost-effective and scalable
 
 # Chart Details
 
-This chart will do the following:
-
-* Rollout victoria metrics cluster
+Note that this chart only installs VictoriaMetrics cluster components such as vminsert, vmselect and vmstorage. It will not create or configure metrics scraping. If you are looking for a chart to configure monitoring stack in cluster check out [victoria-metrics-k8s-stack chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack#helm-chart-for-victoria-metrics-kubernetes-monitoring-stack).
```

Comment:
__Note__:  this chart installs VictoriaMetrics cluster components such as vminsert, vmselect and vmstorage. It doesn't create or configure metrics scraping. If you are looking for a chart to configure monitoring stack in cluster check out [victoria-metrics-k8s-stack chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack#helm-chart-for-victoria-metrics-kubernetes-monitoring-stack).